src/search/landmarks/landmark_factory_h_m.cc

```diff
@@ -588,78 +589,25 @@ void LandmarkFactoryHM::initialize(const TaskProxy &task_proxy) {
     build_pm_ops(task_proxy);
 }
 
-void LandmarkFactoryHM::generate(const TaskProxy &task_proxy) {
-    if (only_causal_landmarks) {
-        Exploration exploration(task_proxy);
-        discard_noncausal_landmarks(task_proxy, exploration);
-    }
-    if (!disjunctive_landmarks)
-        discard_disjunctive_landmarks();
+void LandmarkFactoryHM::postprocess(const TaskProxy &task_proxy) {
```

Comment:
Why is the `only_causal_landmarks` of no interest anymore? According to the reports, this option affects the number of landmarks, doesn't it?

Reply:
Yes, but only because the implementation is buggy :). For m=1 setting the option to true doesn't change the number of landmarks. While I don't know exactly what happens in `is_causal_landmark` with conjunctive landmarks I doubt it's correct and in theory m=2 should also only produce causal landmarks.